modules/apps/message-boards/message-boards-api/src/main/java/com/liferay/message/boards/configuration/MBModerationGroupConfiguration.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.message.boards.moderation.internal.configuration;
```

Comment:
You can't do this. Moderation is a package to cover one specific feature (in fact it was only developed for Questions). If you move this to message-board-api you are integrating the feature completely inside MB and I don't think we want this, this flow is not tested/supported as MB api.